.pipelines/build-pipelines.yml

```diff
@@ -211,7 +211,7 @@ steps:
   inputs:
     command: 'push'
     feedsToUse: 'select'
-    packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg;$(Build.ArtifactStagingDirectory)/**/*.snupkg'
+    packagesToPush: '$(Build.ArtifactStagingDirectory)/**/*.nupkg'
```

Comment:
for internal feed I think we need to do it separately.Can you verify from the logs once. I see the pipeline didn't fail on that step.

Reply:
Just checked Azure private feed do not support symbols package as of now.